schemas/configuration/v5/server_config.json.in

```diff
@@ -67,15 +129,56 @@
                     "uniqueItems": true
                 }
             },
-            "required": ["control_type", "users"]
+            "required": [
+                "control_type",
+                "users"
+            ]
         },
-        "maximum_connections": {
-            "type": "integer",
-            "minimum": 1
+        "default_dir_mode": {
+            "type": "string",
+            "minLength": 4,
+            "maxLength": 4
+        },
+        "default_file_mode": {
+            "type": "string",
+            "minLength": 4,
+            "maxLength": 4
+        },
+        "default_hash_scheme": {
+            "type": "string",
+            "maxLength": 63
+        },
+        "default_resource_name": {
+            "type": "string",
```

Comment:
does this need a minLength?  0?   is that redundant b/c if it's required it has to exist?

Reply:
I don't think so. See https://github.com/irods/irods/pull/8357#discussion_r2041954837.